data/mods/Generic_Guns/gg_ammo_override.json

```diff
@@ -426,4 +426,4 @@
         "count" : 10,
         "effects" : ["INCENDIARY"]
     }
-]
```

Comment:
What file did you open this in and edit it? What it's saying is the that the ending newline character on the last line was removed, which some text editors will do automatically.